pyproject.toml

```diff
@@ -35,13 +35,17 @@ sqlalchemy      = ">=1.3.6, <2"
 pyarrow         = ">=8.0.0"
 
 # Extra dependencies for cell_locations 
-fsspec  =   {version = ">=2023.1.0",    extras=["cell_locations"]}
-s3fs    =   {version = ">=0.4.2",       extras=["cell_locations"]}
-boto3   =   {version = ">=1.26.79",     extras=["cell_locations"]}
-fire    =   {version = ">=0.5.0",       extras=["cell_locations"]}
+fsspec  =   {version = ">=2023.1.0",    optional=true}
+s3fs    =   {version = ">=2023.4.0",       optional=true}
```

Comment:
Double checking on the version update here: would a more recent version be useful (https://pypi.org/project/s3fs/#history)? Or were there reasons for avoiding that? If there are reasons for it, consider making a note here which we can address later.

Reply:
There's no particular reason. I'm just usually conservative in updating dependencies in case there's possible dependency conflicts so I usually pick a version 3-6 months back. That might not be best practice though. It would be good to go through all our dependencies systematically at some point and update them all.

These particular dependencies were updated because of a version resolution bug that caused ancient versions of the packages to be installed. It came up as a warning during a pytest run. See [here](https://github.com/dask/dask/issues/10276)

Reply:
Thanks for the reply - I don't have a strong opinion here, this was mostly a double check. Voicing another perspective; I generally trust the latest release from a package (until there's a reason not to) with the understanding that making the latest release available might grant better performance, etc. to our users. Definitely have been bitten by this sometimes too.